Cache/Main.lean

```diff
@@ -11,31 +11,46 @@ Usage: cache [COMMAND]
 
 Commands:
   # No priviledge required
-  get       Download linked files missing on the local cache and decompress
-  get!      Download all linked files and decompress
-  mk        Compress non-compressed build files into the local cache
-  mk!       Compress build files into the local cache (no skipping)
-  unpack    Decompress linked already downloaded files
-  clean     Delete non-linked files
-  clean!    Delete everything on the local cache
+  get  [ARGS]  Download linked files missing on the local cache and decompress
+  get! [ARGS]  Download all linked files and decompress
+  mk           Compress non-compressed build files into the local cache
+  mk!          Compress build files into the local cache (no skipping)
+  unpack       Decompress linked already downloaded files
+  clean        Delete non-linked files
+  clean!       Delete everything on the local cache
 
   # Privilege required
-  put       Run 'mk' then upload linked files missing on the server
-  put!      Run 'mk' then upload all linked files
-  commit    Write a commit on the server
-  commit!   (Over)write a commit on the server
-  collect   TODO
+  put          Run 'mk' then upload linked files missing on the server
+  put!         Run 'mk' then upload all linked files
+  commit       Write a commit on the server
+  commit!      Overwrite a commit on the server
+  collect      TODO
 
 * Linked files refer to local cache files with corresponding Lean sources
-* Commands ending with '!' should be used manually, when hot-fixes are needed"
+* Commands ending with '!' should be used manually, when hot-fixes are needed
+
+# The arguments for 'get' and 'get!'
+
+'get' and 'get!' can process list of paths containing at most one wildcard each,
+allowing the user to be more specific about what should be downloaded. Example:
+
+$ lake exe cache get Mathlib/Algebra/Field/* Mathlib/Data *Conjugate.lean
```

Comment:
This only works on linux, right? The shell doesn't expand globs on windows.

Reply:
No, it works even without expanded globs. In fact, when I coded it. I didn't expect Linux terminals to do it for me.